### PR TITLE
perf: monitor pdf generation performance

### LINKED ISSUE #246: ci: node version undefined for `pdf-generator` job
# Description

The `pdf-generator` was probably copy-pasted from the `setup-and-test` job, which runs in a matrix; the [problem is that pdf-generator uses an undefined `matrix.node-version`](https://github.com/s1seven/schema-tools/blob/main/.github/workflows/node.yml#L94-L122)

### LINKED ISSUE #247: perf: monitor PDF generation performance
# Description

From our usage of S1Seven's platform, we observed that producing PDF for an average certificate takes 2-3 seconds.
Still, we have yet to determine where most of the time is spent and if the process can be improved.

I suggest running measurements in our example scripts and eventually adding some thresholds in our integration tests.

### PR DESCRIPTION
**Pull request by getlarge**
# Description

- Fixes #247 
- Fixes #246 
- Remove references to previous release manager (`@jscultery/semver`)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by getlarge — COMMENTED** [review]
self review ✔️

File: packages/generate-coa-pdf-template/utils/create-example.ts [hunk 1]
```diff
@@ -12,10 +17,10 @@ import styles from './styles';
 
 const fonts = {
   Lato: {
-    normal: `${__dirname}/../../../node_modules/lato-font/fonts/lato-normal/lato-normal.woff`,
-    bold: `${__dirname}/../../../node_modules/lato-font/fonts/lato-bold/lato-bold.woff`,
-    italics: `${__dirname}/../../../node_modules/lato-font/fonts/lato-light-italic/lato-light-italic.woff`,
-    light: `${__dirname}/../../../node_modules/lato-font/fonts/lato-light/lato-light.woff`,
+    normal: `${__dirname}/../../../node_modules/@fontsource/lato/files/lato-latin-400-normal.woff`,
```
Comment (eamon0989) [inline packages/generate-coa-pdf-template/utils/create-example.ts:20]: Why the change from `lato-font` to `fontsource` just out of interest?

**Review by eamon0989 — APPROVED** [review]
Looking forward to seeing how we can improve the performance. There may be some conflicts with Christoph's PR, especially with the fonts

File: packages/generate-coa-pdf-template/utils/create-example.ts (hunk 1 above)
Comment (getlarge) [reply packages/generate-coa-pdf-template/utils/create-example.ts:20]: https://github.com/s1seven/schema-tools/issues/247#issuecomment-1961890353

File: packages/generate-en10168-pdf-template/project.json [hunk 2]
```diff
@@ -4,21 +4,13 @@
...
-      "dependsOn": ["build"]
-    },
     "build": {
       "executor": "@nx/esbuild:esbuild",
       "outputs": ["{options.outputPath}"],
       "dependsOn": ["create-interfaces"],
       "defaultConfiguration": "production",
       "options": {
-        "outputPath": "packages/generate-en10168-pdf-template/dist",
+        "outputPath": "dist/packages/generate-en10168-pdf-template",
```
[the remark this answers is not in the document]
Comment (getlarge) [reply packages/generate-en10168-pdf-template/project.json:13]: simply to harmonize the location of the build output across all projects

**Review by christophbuehler — DISMISSED** [review]

**Review by eamon0989 — APPROVED** [review]
